### DISCUSSION TITLE
Module not found (css)

### DISCUSSION THREAD

**Discussion by Grisoly** [vercel/community#4658] [Help]
### Summary

Hello, I am following the course [https://nextjs.org/learn/dashboard-app/css-styling](url) and in chapter 2 for CSS styling, when I replace the Tailwind with the CSS module, it gives me this error. I made sure the home.module.css and the page.tsx are saved aftter the replacement but I still get the error. How can we solve this ? 

![Screenshot from 2023-10-29 11-52-56](https://github.com/vercel/community/assets/120173770/0f0ce4d3-87b2-49ca-a9e5-7b16a3a511d0)

### Steps to Reproduce

> Inside /app/ui you would create a new file called home.module.css and add the following CSS rules:

`.shape {
  height: 0;
  width: 0;
  border-bottom: 30px solid black;
  border-left: 20px solid transparent;
  border-right: 20px solid transparent;
}`

> Then, inside your /app/page.tsx file import the styles and replace the tailwind class names from the div, and pass styles.shape instead:

` import styles from './home.module.css';`
` //...`
`<div className={styles.shape}></div>;`

> Save your changes and preview them in the browser. You should see the same shape as before.

**Comment by jeremiemv** [comment]
@Grisoly this got fixed.

It now mentions `import styles from '@/app/ui/home.module.css';`

**Reply by Grisoly** [reply]
Awesome, thanks @jeremiemv 💯
